app/schemas/models/level_component.coffee

```diff
@@ -106,8 +106,6 @@ PropertyDocumentationSchema = c.object {
         title: 'Variable Name'
         description: 'Variable name this property is autocompleted into.'
         default: 'result'
-  section: c.shortString(title: 'Methods Bank Section')
-  subSection: c.shortString(title: 'Methods Bank Sub-Section')
```

Comment:
I believe I heard that these were not used anywhere. Just checking that assumption.

Reply:
Yes, I added it earlier to test but now removing since its not required. And also removed using it from the test level.